data/interim/france/france_confirmed_cases_by_region.csv

```diff
@@ -26,13 +26,18 @@ date,adm1_name,cum_confirmed_cases,adm0_name,adm1,adm1_pop,cum_confirmed_cases_i
 3/10/2020,"Guadeloupe",0,"France",1,376879,0

```

Comment:
Seems like `data/interim/france/france_confirmed_cases_by_region.csv` changed to incorporate new data. Can we also include the raw data that was needed to create this file in this PR?

Reply:
already uploaded separately by Jeanette

Reply:
👍